packages/react/src/components/fileInput/FileInput.stories.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { boolean, number, object, radios, text } from '@storybook/addon-knobs';
+
+import { FileInput } from './FileInput';
+
+export default {
+  component: FileInput,
+  title: 'Components/FileInput',
+  decorators: [(storyFn) => <div style={{ maxWidth: '400px' }}>{storyFn()}</div>],
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+export const Single = (args) => {
+  const [file, setFile] = React.useState<File[]>();
+  // eslint-disable-next-line no-console
+  console.log('selected file', file);
+
+  return (
+    <FileInput
+      {...args}
+      id="file-input"
+      label="Choose a file"
+      buttonLabel="Add file"
```

Comment:
Not sure if this would grammatically require an article 'a'? Though not a big deal anyways

Reply:
- [ ] Fix typo